src/pgwire-common/src/conn.rs

```diff
@@ -20,6 +20,7 @@ use tokio_postgres::error::SqlState;
 use crate::ErrorResponse;
 
 pub const CONN_UUID_KEY: &str = "mz_connection_uuid";
+pub const MZ_FORWARD_FOR_KEY: &str = "mz_forward_for";
```

Comment:
```suggestion
pub const MZ_FORWARDED_FOR_KEY: &str = "mz_forwarded_for";
```

(to match the HTTP header)